apps/vr-tests/src/stories/NewSlider.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import Screener from 'screener-storybook/src/screener';
+import { storiesOf } from '@storybook/react';
+import { FabricDecoratorTall } from '../utilities';
+import { Slider } from '@uifabric/experiments';
+
+const marks = [
+  { value: 0, label: '0' },
+  { value: 40, label: '40' },
+  { value: 60, label: '60' },
+  { value: 100, label: '100' }
+];
+
+// tslint:disable:jsx-ban-props
+storiesOf('Slider', module)
```

Comment:
Should this be "new slider", or "experimental slider". otherwise there is a small chance of conflict with the current slider visual tests